src/components/Footer.jsx

```diff
@@ -0,0 +1,204 @@
+import React from "react";
+import styled from "styled-components";
+import { Discord } from "./icons/Vector-Discord.js";
+import { Twitter } from "./icons/Vector-Twitter.js";
+import theme, { breakpoints } from "../../utils/theme";
+import dclLogo from "../images/logo-dcl.svg";
+import { Instagram } from "./icons/Vector-Instagram.js";
+
+const Footer = (props) => {
+  const pages = [
+    {
+      name: "TERMS & SERVICES",
+      url: "https://decentraland.org/terms/",
+    },
+    {
+      name: "PRIVACY POLICIES",
+      url: "https://decentraland.org/privacy/"
+    }
+  ];
+
+  const socialLinks = [
+    {
+      name: "Twitter",
+      file: Twitter,
+      url: "https://twitter.com/decentraland",
+    },
+    {
+      name: "Instagram",
+      file: Instagram,
+      url: "https://www.instagram.com/decentraland_foundation/",
+    },
+    {
+      name: "Discord",
+      file: Discord,
+      url: "https://decentraland.org/discord",
+    },
+  ];
+  
+  return (
+    <StyledFooter>
+      <StyledLogoContainer>
+        <a href="https://play.decentraland.org/" target="_blank">
+          <StyledLogo alt={"Decentraland Logo"} src={dclLogo} />
+        </a>
+      </StyledLogoContainer>
+      <StyledLinks>
+        <StyledLinksList>
+          {pages.map((page) => (
+            <PageLink key={page.name}>
+              <a target="_blank" href={page.url}>
```

Comment:
Please, include the `rel="noopener"` in all the links:
```suggestion
              <a target="_blank" rel="noopener" href={page.url}>
```